commit ab2f85e5969379726c88a1dda24aab12f7cee5bb
Author: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date:   Sat Dec 29 13:57:09 2018 +0100

    ARM: dts: meson8b: odroidc1: mark the SD card detection GPIO active-low
    
    [ Upstream commit 3fb348e030319f20ebbde082a449d4bf8a96f2fd ]
    
    After commit 89a5e15bcba87d ("gpio/mmc/of: Respect polarity in the device
    tree") SD cards are not detected anymore.
    
    The CD GPIO is "active low" on Odroid-C1. The MMC dt-bindings specify:
    "[...] using the "cd-inverted" property means, that the CD line is active
    high, i.e. it is high, when a card is inserted".
    
    Fix the description of the SD card by marking it as GPIO_ACTIVE_LOW and
    drop the "cd-inverted" property. This makes the definition consistent
    with the existing dt-bindings and fixes the check whether an SD card is
    inserted.
    
    Fixes: e03efbce6bebf5 ("ARM: dts: meson8b-odroidc1: add microSD support")
    Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
    Reviewed-by: Linus Walleij <linus.walleij@linaro.org>
    Tested-by: Anand Moon <linux.amoon@gmail.com>
    Signed-off-by: Kevin Hilman <khilman@baylibre.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/meson8b-odroidc1.dts b/arch/arm/boot/dts/meson8b-odroidc1.dts
index 9e48936e2d8d..8fdeeffecbdb 100644
--- a/arch/arm/boot/dts/meson8b-odroidc1.dts
+++ b/arch/arm/boot/dts/meson8b-odroidc1.dts
@@ -171,8 +171,7 @@
 		cap-sd-highspeed;
 		disable-wp;
 
-		cd-gpios = <&gpio CARD_6 GPIO_ACTIVE_HIGH>;
-		cd-inverted;
+		cd-gpios = <&gpio CARD_6 GPIO_ACTIVE_LOW>;
 
 		vmmc-supply = <&tflash_vdd>;
 		vqmmc-supply = <&tf_io>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1065_ab2f85e5969379726c88a1dda24aab12f7cee5bb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1065_ab2f85e5969379726c88a1dda24aab12f7cee5bb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
